packages/browser/src/storage/BrowserFileSystem.ts

```diff
@@ -0,0 +1,90 @@
+import { BacktraceAttachment, BacktraceStringAttachment, FileSystem } from '@backtrace/sdk-core';
+
+const PREFIX = 'backtrace__';
+
+export class BrowserFileSystem implements FileSystem {
```

Comment:
This suppose to be an example, however we're including it in the library code. Are you sure about it?